bin/state

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# Run Terraform commands on the create_state_bucket project in the Hashicorp docker container,
+# passing our environment variables
+
+docker run --rm -it \
+    -u "$(id -u)" \
+    -v "${PWD}/src/aws/create_state_bucket:/app" \
+    -w "/app" \
+    --env-file ./env.d/aws \
+    "hashicorp/terraform:light" \
+    "$@"
```

Comment:
Why not adding a terraform service in the docker-compose file and add more make rules? or at least use a terraform docker-compose service here?